src/views/Organisations/Organisation.vue

```diff
@@ -183,21 +196,224 @@
         </v-fade-transition>
       </v-col>
     </div>
+    <!-- Edit existing organisation -->
```

Comment:
If possible can Edit organisation be a separate component, giving the complexity and size of the file.

Reply:
Having had  a look I don't think that this could easily be done.

Reply:
Created ticket for the task: https://github.com/FAIRsharing/fairsharing.github.io/issues/2267